src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -622,9 +631,9 @@ type Get(nameChain: string) =
     | OneWayToSource _ -> GetError.OneWayToSource |> Error
     | OneWaySeq { Values = vals } -> vals.GetCollection () |> Ok
     | Cmd cmd -> cmd |> box |> Ok
-    | SubModel { Vm = vm } -> vm.Value |> ValueOption.toObj |> box |> Ok
-    | SubModelWin { VmWinState = vm } ->
-        vm.Value
+    | SubModel { GetVm = getvm } -> getvm() |> ValueOption.toObj |> box |> Ok
+    | SubModelWin { GetVmWinState = getvm } ->
+        getvm()
```

Comment:
Nit: keep on one line

Reply:
Like this?

Reply:
I think so. I don't see this comment associated with a current line, but I don't see a second problem in the current code either.